tests/adapters/test_parameter_tree.py

```diff
@@ -247,13 +248,13 @@ def __init__(self):
         }
         self.nested_tree = ParameterTree(self.nested_dict)
 
-        self.callback_tree = deepcopy(self.nested_tree)
-        self.callback_tree.add_callback('branch/', self.branch_callback)
+        # self.callback_tree = deepcopy(self.nested_tree)
```

Comment:
References to deprecated callback mechanism can be removed from tests rather than commented out, now callback is removed from `ParameterTree` itself